### ISSUE TITLE
Memory problems caused by graph displays

### ISSUE THREAD

**Issue by Daniel-Mietchen** [fnielsen/scholia#262]
Scholia pages with graph queries like the Co-occurring topics graph tend to slow down the browser (at least Firefox and Safari under OSX) considerably for complex graphs, to the point that the tab becomes almost unusable. The problem usually persists even after the tab is closed, so it seems there is something wrong with memory cleanup.

The issue has been around since the graph queries have been introduced, but it is turning into a problem now, so I'm opening this ticket for it.

**Comment by coderwassananmol** [comment]
This issue still persists. The web page slows down so much that it takes forever to load.

**Comment by fnielsen** [comment]
Yes, there are some graph displays where this is an issue. I wonder if you, @coderwassananmol , have any specific problem?

**Comment by Daniel-Mietchen** [comment]
There is a range of test cases in #615  .
